01.md

```diff
@@ -85,10 +85,11 @@ As a convention, all single-letter (only english alphabet letters: a-z, A-Z) key
 
 ### Kinds
 
-Kinds specify how clients should interpret the meaning of each event and the other fields of each event (e.g. an `"r"` tag may have a meaning in an event of kind 1 and an entirely different meaning in an event of kind 10002). Each NIP may define the meaning of a set of kinds that weren't defined elsewhere. This NIP defines two basic kinds:
+Kinds specify how clients should interpret the meaning of each event and the other fields of each event (e.g. an `"r"` tag may have a meaning in an event of kind 1 and an entirely different meaning in an event of kind 10002). Each NIP may define the meaning of a set of kinds that weren't defined elsewhere. [NIP-10](10.md), for instance, especifies the `kind:1` text note for social media applications.  
+
+This NIP defines one basic kind:
 
 - `0`: **metadata**: the `content` is set to a stringified JSON object `{name: <username>, about: <string>, picture: <url, string>}` describing the user who created the event. A relay may delete older events once it gets a new one for the same pubkey.
-- `1`: **text note**: the `content` is set to the **plaintext** content of a note (anything the user wants to say). Content that must be parsed, such as Markdown and HTML, should not be used. Clients should also not parse content as those.
```

Comment:
```md
#### Text Notes (Kind 1)

Social media is a core use-case of Nostr, and kind 1 text notes are the way to make a basic "post". See (NIP-10)[./NIP-10.md] for the full specification of text notes.

#### Other Stuff

Nostr can support many other use-cases by using additional kinds. See the [full list of kinds] (./README.md#event-kinds) or consider proposing your own.
```

Reply:
> #### Other Stuff

I also don't like this separation. It creates a sense of second-class citizen if you are not a kind1 client/dev. Again, it's already in the name, there is no need to repeat it everywhere.